animus-3/source/animus/Animus.cpp

```diff
@@ -230,10 +229,6 @@ void CAnimus::MillisLoop()
     PreviousMillis = CurrentMillis;
     ReadyMillis = true;
     RefreshCounter++;
```

Comment:
Since we are removing AsyncRefresh entirely, is there still a point to keeping RefreshCounter? Perhaps we should remove it entirely.

Reply:
Nope, no reason at all- let's throw it out